README.md

```
@@ -64,13 +64,52 @@ To install and activate a new extension you have to:
+### 3. Testing Specific Production/PR Tags Locally (`IMAGE_TAG`)
+The configuration reads the environment dynamically. If you want to simulate a specific release version (e.g., `1.23.4`) or pull request layout locally without running the online GitHub Actions pipeline, prefix your command with the `IMAGE_TAG` variable:
+
+```bash
+IMAGE_TAG=1.23.4 docker buildx bake --load
+```
+
+# Pushing Local ARM64 Builds to GHCR (Manual)
+
+If you need to share your locally compiled Apple Silicon (`arm64`) images on GitHub Packages manually, you can bake and push them using the `IMAGE_TAG` environment variable.
+
+### 1. Authenticate with GHCR via GitHub CLI (`gh`)
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)